symbolic-debuginfo/src/wasm.rs

```diff
@@ -131,7 +131,12 @@ impl<'data> WasmObject<'data> {
 
     /// Determines whether this object contains embedded source.
     pub fn has_sources(&self) -> bool {
-        false
+        match self.debug_session() {
+            Ok(session) => session
+                .files()
+                .any(|f| f.is_ok_and(|f| f.source_str().is_some())),
+            Err(_) => false,
+        }
```

Comment:
In `elf.rs` you write this in `let-else` style. Either option is fine, but I think it should be uniform.